resources/css/administration.css

```diff
@@ -61,6 +61,10 @@ h4 {
     height: 55vh;
 }
 
+:fullscreen .wt-location-edit-map {
+    height: 100% !important;
```

Comment:
I don't think the `!important` is necessary (we are not trying to override another rule with a higher specificity).
It seems to work without it - unless I've missed something.

Reply:
copy & paste!!